lib/daru/index/multi_index.rb

```diff
@@ -17,12 +17,10 @@ def levels
     def initialize opts={}
       labels = opts[:labels]
       levels = opts[:levels]
-      raise ArgumentError,
-        'Must specify both labels and levels' unless labels && levels
-      raise ArgumentError,
-        'Labels and levels should be same size' if labels.size != levels.size
-      raise ArgumentError,
-        'Incorrect labels and levels' if incorrect_fields?(labels, levels)
+
+      raise ArgumentError, 'Must specify both labels and levels' unless labels && levels
+      raise ArgumentError, 'Labels and levels should be same size' if labels.size != levels.size
```

Comment:
Technically no. But its not a hard rule and can be relaxed if its going slightly more than 80 and readability is enhanced as a result.